src/config/routes.ts

```diff
@@ -0,0 +1,60 @@
+import IRoutes from '../interfaces/routes';
+import Agents from '../pages/agents';
+import Beneficiary from '../pages/beneficiary/index';
+import Dashboard from '../pages/dashboard';
+import Location from '../pages/location';
+import Occupation from '../pages/occupation';
+import Users from '../pages/users';
+
+const routes: IRoutes[] = [
+  {
+    path: '/dashboard',
+    name: 'Dashboard Page',
+    component: Dashboard,
+    exact: true,
+  },
+  {
+    path: '/beneficiary/index',
+    name: 'Beneficiary Page',
+    component: Beneficiary,
+    exact: true,
+  },
+  {
+    path: '/location',
+    name: 'Location Page',
+    component: Location,
+    exact: true,
+  },
+  {
+    path: '/occuoation',
```

Comment:
`/occupation` right?

Reply:
Yes, will resolve and push ALL changes with incoming PR

Reply:
Yes, will resolve and push ALL changes with incoming PR